x-pack/plugins/rule_registry/common/assets/field_maps/technical_rule_field_map.ts

```diff
@@ -17,6 +17,7 @@ export const technicalRuleFieldMap = {
     Fields.TAGS
   ),
   [Fields.ALERT_RULE_PARAMS]: { type: 'keyword', index: false },
+  [Fields.ALERT_RULE_PARAMETERS]: { type: 'flattened' },
```

Comment:
In the rule mappings, we had to add `ignore_above` to allow indexing of super long string values => https://github.com/elastic/kibana/blob/main/x-pack/plugins/alerting/server/saved_objects/mappings.json#L54 otherwise the indexing failed.